provider-blackduck/src/main/java/com/synopsys/integration/alert/provider/blackduck/BlackDuckProviderDataAccessor.java

```diff
@@ -257,16 +257,18 @@ public void updateProjectAndUserData(Long providerConfigId, Map<ProviderProject,
 
         Predicate<UserView> searchFilter = userView -> StringUtils.isNotBlank(userView.getEmail());
         if (StringUtils.isNotBlank(searchTerm)) {
-            searchFilter = searchFilter.and(userView -> StringUtils.containsIgnoreCase(userView.getEmail(), searchTerm));
+            Predicate<UserView> emailMatches = userView -> StringUtils.containsIgnoreCase(userView.getEmail(), searchTerm);
+            Predicate<UserView> usernameMatches = userView -> StringUtils.containsIgnoreCase(userView.getUserName(), searchTerm);
+
+            searchFilter = searchFilter.and(emailMatches.or(usernameMatches));
```

Comment:
@gkillough I broke it up into predicate pieces-- I feel like there's something more I could do but I'm not seeing it. Let me know what you think!

Reply:
I think it's good there. You could go further, but it might make it too convoluted. I just wanted it to be a bit more readable and I think that's accomplished.